src/Microsoft.AspNetCore.Mvc.Abstractions/IUrlHelper.cs

```diff
@@ -16,11 +16,13 @@ public interface IUrlHelper
         ActionContext ActionContext { get; }
 
         /// <summary>
-        /// Generates a fully qualified or absolute URL specified by <see cref="UrlActionContext"/> for an action
-        /// method, which contains action name, controller name, route values, protocol to use, host name, and fragment.
+        /// Generates a URL with an absolute path for an action method, which contains the action
+        /// name, controller name, route values, protocol to use, host name, and fragment specified by
+        /// <see cref="UrlActionContext"/>. Generates an absolute URL if <see cref="UrlActionContext.Protocol"/> and
+        /// <see cref="UrlActionContext.Host"/> are non-<c>null</c>.
         /// </summary>
         /// <param name="actionContext">The context object for the generated URLs for an action method.</param>
-        /// <returns>The fully qualified or absolute URL to an action method.</returns>
+        /// <returns>The URL with an absolute path for an action method.</returns>
```

Comment:
I'm struggling with the "otherwise" nature of your suggestion. URLs with absolute paths are a strict subset of absolute URLs. Will play w/ the wording a bit more ...